my_game.py

```diff
@@ -155,13 +155,19 @@ def on_update(self, delta_time):
 
 class Asteroid(arcade.Sprite):
 
-    def __init__(self):
+    valid_sizes = {1: "images/Meteors/meteorGrey_tiny1.png", 2: "images/Meteors/meteorGrey_small1.png", 3: "images/Meteors/meteorGrey_med1.png", 4: "images/Meteors/meteorGrey_big1.png"}
+    
+    
+    def __init__(self, size=None):
 
         # Initialize the asteroid
-
+        
+        if size is None:
+            size = random.choice(list(Asteroid.valid_sizes.keys()))
+        
```

Comment:
Since _size_ is not saved (no _self.size_), how will you later know if the Asteroid is to be split into smaller Asteroids when hit?